app/controllers/application_controller.rb

```diff
@@ -471,11 +471,23 @@ def fetch_translations
     WebTranslateIt.fetch_translations
   end
 
+  def redirect_https?
+    always_use_ssl = APP_CONFIG.always_use_ssl.to_s.downcase == "true"
+    return !request.ssl? && always_use_ssl
```

Comment:
You can remove the `return` here. Last expression in the method is always returned so the Ruby way is not to add `return` in this case.